packages/cli/test/scripts/init.test.js

```diff
@@ -6,18 +6,27 @@ import { cleanup, cleanupfn } from '../helpers/cleanup'
 
 import init from '../../src/scripts/init'
 import ZosPackageFile from '../../src/models/files/ZosPackageFile'
+import ZosConfig from '../../src/models/config/ZosConfig'
 
 contract('init script', function() {
   const name = 'MyApp';
   const version = '0.3.0';
   const tmpDir = 'test/tmp';
 
-  before('create tmp dir', () => fs.createDir(tmpDir))
-  after('cleanup tmp dir', cleanupfn(tmpDir))
+  before('create tmp dir', function () {
+    fs.createDir(tmpDir)
+    this.zosConfigInitialize = ZosConfig.prototype.initialize;
```

Comment:
Why not sinon? (agree on leaving this like this though, no need to change)

Reply:
no strong reason, but I prefer to use sinon. Nice catch.